packages/components/psammead-radio-schedule/src/StartTime/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,269 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`StartTime should render LTR correctly 1`] = `
+.c2 {
+  vertical-align: middle;
+  margin: 0 0.25rem;
+  color: #222222;
+  fill: currentColor;
+  width: 0.725rem;
+  height: 0.725rem;
+}
+
+.c4 {
+  font-size: 0.875rem;
+  line-height: 1.125rem;
+  color: #6E6E73;
+  display: block;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.c0 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.c1 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  padding-right: 0.25rem;
+  padding-left: 0.5rem;
+}
+
+.c1 > svg {
+  color: #5A5A5A;
```

Comment:
Is this going to need fill: currentColor; as well??

Reply:
Technically svg should already have `fill: currentColor` (https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-assets/src/svgs/coreIcons.jsx#L24) . But I've noticed that in some cases e.g. in Bulletin we add that value again when we are using the icon. So I can add this property again if needed?

Reply:
I can't say if it's needed without testing it...

Reply:
I checked the icon in developer tools and the svg has property `fill: currentColor`

<img width="188" alt="Screenshot 2020-01-28 at 11 28 03" src="https://user-images.githubusercontent.com/17802955/73260203-580d9480-41c1-11ea-8d34-b3c8b15d1a64.png">

It is also in the snapshot file here: https://github.com/bbc/psammead/blob/dbd13f5a5582ec41740e0307d933c5818054dab4/packages/components/psammead-radio-schedule/src/StartTime/__snapshots__/index.test.jsx.snap#L8